jax/experimental/jax2tf/tests/primitives_test.py

```diff
@@ -572,6 +572,19 @@ def custom_assert(result_jax, result_tf):  # noqa: F811
   def test_round(self, harness: primitive_harness.Harness):
     self.ConvertAndCompare(harness.dyn_fun, *harness.dyn_args_maker(self.rng()))
 
+  @primitive_harness.parameterized(primitive_harness.lax_convert_element_type)
+  def test_convert_element_type(self, harness: primitive_harness.Harness):
+    old_dtype = harness.params["old_dtype"]
```

Comment:
I think it would be best to try to move this filtering in the construction of the harnesses, perhaps also with a comment about the nature of the bug.